committeeoversightapp/templatetags/committeeoversight_tags.py

```diff
@@ -0,0 +1,32 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter(name='lookup')
```

Comment:
are you using this anyplace else? if not, i think it's safe to remove!

Reply:
yes!